src/main/java/io/keyko/monitoring/examples/basic/BasicStreamManager.java

```diff
@@ -18,7 +24,19 @@ public BasicStreamManager(StreamerConfig config) {
   }
 
   @Override
-  protected void processStreams(KStream<String, EventRecord> eventStream, KStream<String, ViewRecord> viewStream,  KTable<String, BlockRecord> blockTable) {
+  protected void processStreams(KStream<String, EventRecord> eventStream, KStream<String, ViewRecord> viewStream, KStream<String, LogRecord> logStream, KTable<String, BlockRecord> blockTable) {
+
+    String ethScan = configuration.getEtherscanUrl();
+    String contractAbiParams = configuration.getEtherscanGetContract();
+    String url = ethScan.concat(contractAbiParams);
+
+    KStream<String, EventRecord> eventFromLogStream = Transformations.transformLogToEvent(logStream,
+                                                                          url,
+                                                                          configuration.getEtherscanApikey(),
+                                                                          configuration.getEtherscanSendNotMatchToTopic(),
+                                                                          configuration.getLogNotMatchErrorTopic());
+
+    eventFromLogStream.to("w3m-events-from-log", Produced.with(Serdes.String(), Web3MonitoringSerdes.getEventSerde()));
```

Comment:
I guess the topic should come from configuration

Reply:
it's just an example of how to use it, not part of the framework